sputnikdao2/src/upgrade.rs

```diff
@@ -25,13 +26,30 @@ pub struct FactoryInfo {
     pub auto_update: bool,
 }
 
+pub fn get_default_factory_id() -> AccountId {
```

Comment:
Yes, on testnet this will work only with DAOs created from the official testnet factory. It can't know about the custom one that you've created.

Reply:
@ctindogaru i believe it could actually. Since every DAO is just a sub-account of the factory, the string would need to be split and then you have the factory. Thoughts?

Reply:
I think now it should work with everything. Brilliant suggestion mister 👍